meson.build

```diff
@@ -137,27 +134,31 @@ if not meson.is_subproject() and get_option('build_library')
 		filebase: 'substrate',
 		libraries: [libSubstrate, deps],
 		version: meson.project_version(),
+		extra_cflags: libSubstrateArgs,
 		description: 'A collection of headers containing useful tools and gadgets for building C++ programs'
 	)
 endif
 
-variables = {
-	'compile_args': '-I@0@'.format(meson.current_source_dir()),
-}
-
 if targetLibraryBuildable
-	variables += {
-		'link_args': libSubstrate.full_path(),
-	}
+	substrate_dep = declare_dependency(
```

Comment:
These *must* declare `variables: ` as before as other projects depend on those being supplied via the dependency object. Can't make `crunch` do the right thing otherwise due to Meson limitations